packages/codemirror/src/mimetype.ts

```diff
@@ -51,6 +51,9 @@ class CodeMirrorMimeTypeService implements IEditorMimeTypeService {
    * If a mime type cannot be found returns the defaul mime type `text/plain`, never `null`.
    */
   getMimeTypeByFilePath(path: string): string {
+    if (path.split('.').slice(-1)[0] === 'ipy') {
```

Comment:
Do you mind using [PathExt.extname](https://github.com/jupyterlab/jupyterlab/blob/77ac304d2633ce937c00e506b8e827e0c4de2f70/packages/coreutils/src/path.ts#L64) for this?